docker/setup_dashboard-docker.sh

```diff
@@ -53,10 +53,24 @@ cd NetSageNavigation
 make install
 cd ..
 
-#Install slope graph plugin
-cd Netsage-Slope_graph/
+#Install slope graph
+# HACK: for some reason this fails to install unless it's moved to a different location.
+cp -r  Netsage-Slope_graph /tmp/
+pushd . 
+cd /tmp/Netsage-Slope_graph/
 make install
-cd ..
+popd
+
+#Install discipline map
+# HACK: for some reason this fails to install unless it's moved to a different location.
+cp -r  science-discipline-map-plugin /tmp/
+pushd . 
+cd /tmp/science-discipline-map-plugin/
+make install
+popd
+
```

Comment:
you don't need to copy to /tmp any longer.  If you compare to master you can see an example of how to do this.